python/triton/language/core.py

```diff
@@ -378,7 +378,7 @@ def __init__(self, name):
                 self.primitive_bitwidth = 32
                 self.exponent_bias = 127
             elif name == 'fp64':
-                self.fp_mantissa_width = 53
+                self.fp_mantissa_width = 52
```

Comment:
I'm guessing the reason is that from precision point of view there is still 53 bits as per wikipedia:
```
The format is written with the [significand](https://en.wikipedia.org/wiki/Significand) having an implicit integer bit of value 1 (except for special data, see the exponent encoding below). With the 52 bits of the fraction (F) significand appearing in the memory format, the total precision is therefore 53 bits (approximately 16 decimal digits, 53 log10(2) ≈ 15.955). The bits are laid out as follows:
```

Reply:
This still wouldn't be consistent with the values for the other floating point types, e.g. we have fp32's mantissa width specified above as 23 bits and not 24

Reply:
right, fair point. Maybe this is just a mistake then